apps/src/templates/Button.jsx

```diff
@@ -114,7 +126,16 @@ const styles = {
       paddingLeft: 30,
       paddingRight: 30,
       lineHeight: '40px'
-    }
+    },
+    [ButtonSize.mega]: {
+      fontSize: 27,
+      fontFamily: '"Gotham 7r", sans-serif',
+      height: 70,
+      paddingLeft: 30,
+      paddingRight: 30,
+      lineHeight: '70px',
+      width: 310,
+    },
```

Comment:
New colors/sizes of buttons should prob get signoff from Mark and/or other design minded folks.

Reply:
Is it possible to make the caret icon more bold too? Otherwise buttons LGTM

Reply:
Mark gave a thumbs up for the size, but decided we don't need it to be teal.  That way all of the buttons on the page (except submit) will consistently gray. Since the only difference is size, it doesn't seem like it warrants making a whole separate, yet very similar, component.

Reply:
We're changing a lot more elements of the style for our "mega" than for our others (i.e. fontSize, fontFamily, width).

Do we think we're going to be using this "mega" size in other places, and will we always want all these props? I wonder if we should instead be using the `style` for these.